### PR TITLE
Add HIT policy

### PR DESCRIPTION
**Pull request by destroy314**
## What this does
The Humanoid Imitation Transformer (HIT) is derived from the paper HumanPlus: Humanoid Shadowing and Imitation from Humans (http://arxiv.org/abs/2406.10454), and it is a modification of the ACT model. The key changes include the removal of the VAE and the use of a BERT-style transformer to predict actions and future image embeddings. Although HIT is originally designed for controlling humanoid robots, it can perform better than ACT in the sim ALOHA environment.

## How it was tested
- Trained with `hit.yaml` on `lerobot/aloha_sim_insertion_human` dataset
- Modified `tests/test_available.py` by importing new HITPolicy class

## How to checkout & try? (for the reviewer)
```bash
$ python lerobot/scripts/eval.py -p destroy314/act_aloha_sim_insertion_human eval.n_episodes=100 eval.batch_size=10
{'avg_sum_reward': 261.26, 'avg_max_reward': 2.35, 'pc_success': 15.0, 'eval_s': 439.620165348053, 'eval_ep_s': 4.396201655864716}

$ python lerobot/scripts/eval.py -p destroy314/hit_aloha_sim_insertion_human eval.n_episodes=100 eval.batch_size=10
{'avg_sum_reward': 261.56, 'avg_max_reward': 2.73, 'pc_success': 28.999999999999996, 'eval_s': 439.3340971469879, 'eval_ep_s': 4.3933409762382505}
```
For training, use
```bash
python lerobot/scripts/train.py policy=hit env=aloha env.task=AlohaInsertion-v0 dataset_repo_id=lerobot/aloha_sim_insertion_human
```
Note that the hyperparameters used for HIT differ from those in the original paper and are instead similar to the ones used in ACT.

### CODE REVIEW

**Comment by destroy314** [comment]
I might have forgotten to call `feat_future.detach()`
